src/Common/Bridge/BridgeFacadeMethodsRule.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace ArchitectureSniffer\Common\Bridge;
+
+use ArchitectureSniffer\SprykerAbstractRule;
+use PHPMD\AbstractNode;
+use PHPMD\Node\ClassNode;
+use PHPMD\Node\MethodNode;
+use PHPMD\Rule\ClassAware;
+use ReflectionMethod;
+use ReflectionType;
+
+class BridgeFacadeMethodsRule extends SprykerAbstractRule implements ClassAware
+{
+    /**
+     * @var string
+     */
+    protected const RULE = 'The bridge facade method must follow CRUD signature';
```

Comment:
```suggestion
    protected const RULE = 'The bridge facade method must follow CRUD signature based on the naming convention';
```
This only affects those with the naming convention correct?
Not all bridge facade methods

Reply:
@gechetspr Or how could we clarify this?

Reply:
yes